### ISSUE TITLE
experiments in notebook

### ISSUE THREAD

**Issue by JohnsonClayton** [JohnsonClayton/DeepLearning-DNSoverHTTPS#6]
create a jupyter notebook that can easily be used to execute necessary experiments to collect accuracy performance with different numbers of features. The list of models needed is below:  
 - Keras-TensorFlow
 - Fast.ai
 - Decision Tree
 - Logistic Regression 
 - Linear Discriminant Analysis
 - k-Nearest Neighbors
 - Naive Bayes 
 - Support Vector Machines
 - Random Forest
 - AdaBoost
 - XGBoost  

Once these experiments are executed, add the results to the README in tabular or graphical format
